.ci/travis_script.sh

```diff
@@ -4,6 +4,8 @@ set -e -x
 
 if [[ -n ${TOXENV} ]]; then
   tox -e ${TOXENV}
+elif [[ "${BIGCHAINDB_DATABASE_BACKEND}" == mongodb ]]; then
+    pytest -v --database-backend=mongodb --cov=bigchaindb
```

Comment:
Some slightly odd tab formatting here; do we normally use 4 tabs for shell scripts?